### PR TITLE
Get things working

### PR DESCRIPTION
**Pull request by Psykar**
This actually works pretty good now!
Might need to look at an opensuse image soonish though...

If you're not keen on maintaining this let me know, I'll hard fork and rename instead.

### CODE REVIEW

**Comment by bigfoot90** [comment]
Nice work!

File: docker-compose.override.yml.dist [hunk 1]
```diff
@@ -1,9 +1,17 @@
 version: '3.4'
 
+volumes:
```
[the remark this answers is not in the document]
Comment (Psykar) [reply docker-compose.override.yml.dist]: Heh, if we move this the override yaml is not needed :)

File: README.md [hunk 2]
```diff
@@ -1,17 +1,22 @@
 # Run Rockstor inside a Docker container
 
 > This is still WORK IN PROGRESS
-
-Unfortunately Rockstor was not meant for running inside a Docker container, but this is a first small step in the correct direction.
-The image builds correcly and all services are running, but the WebUI hans after initial administration account setup.
-If someone has more in depth knownledge of Rockstor, any help wellcome.
+> WARNING: If your host's filesystem is btrfs starting this docker container will enable quotas on your host's filesystem.
 
 Make your own `docker-compose.overide.yml` from `docker-compose.overide.yml.dist`
```
Comment (bigfoot90) [inline README.md]: This file has been deleted

File: Dockerfile [hunk 3]
```diff
@@ -5,10 +5,29 @@ COPY Rockstor.repo ZeroMQ.repo /etc/yum.repos.d/
...
+    yum install -y https://centos7.iuscommunity.org/ius-release.rpm
+RUN yum install -y rockstor git gcc gcc-c++ make python-devel zeromq-devel postgresql-devel file which grubby openssl man postgresql-server postgresql-contrib python-distro postgresql python-setuptools NetworkManager cryptsetup e2fsprogs
+# Because we're going to use git
+RUN yum remove -y rockstor docker-common
+RUN yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+RUN yum install -y docker-ce docker-ce-cli containerd.io
 
+RUN mv /opt/rockstor /opt/rockstor-back
+RUN git clone https://github.com/rockstor/rockstor-core /opt/rockstor
 WORKDIR /opt/rockstor
```
Comment (bigfoot90) [inline Dockerfile]: Is `WORKDIR` needed here?
Can be placed on at the end of file?

**Comment by bigfoot90** [comment]
@Psykar You have done a great work!
I've merged your PR, so you can now rebase your fork on top of this.
I'll close this repo when you're done.

**Pull request merged**